Automattic/jetpack
# Jetpack Setup Wizard: Add Income Question Page

#### Changes proposed in this Pull Request:

This PR builds on the work in https://github.com/Automattic/jetpack/pull/15429 by adding the income question page.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Project: p1HpG7-8BF-p2
Design: pbtFPC-ol-p2

Figma design:
<img width="920" alt="image" src="https://user-images.githubusercontent.com/42627630/80252949-f8971780-863d-11ea-9653-099cc8bc7481.png">

Current state:
<img width="1190" alt="image" src="https://user-images.githubusercontent.com/42627630/80253154-5d527200-863e-11ea-8794-5a4d704d97e8.png">

Figma design mobile:
<img width="295" alt="image" src="https://user-images.githubusercontent.com/42627630/80253116-490e7500-863e-11ea-9d97-35e93c7d5f8b.png">

Current state mobile:
<img width="584" alt="image" src="https://user-images.githubusercontent.com/42627630/80384016-f4e7d880-8869-11ea-9f8d-c8396b9e2b61.png">


#### Testing instructions:
1. Add the following to the end of wp-config.php: add_filter( 'jetpack_show_setup_wizard', '__return_true' );
2. Visit the Jetpack plugin and visit the `/setup` page.
3. Click either "Personal Use" or "Business Use". Verify that you advance to the `/setup/income` page.
4. Verify that back/forward works between these two pages.
5. Verify that you can select various choices both with the mouse and by tabbing with the keyboard.
6. Shrink the window to <= 660px. Verify that the page adjusts according to the design.
7. Verify that you can expand and hide the details.
8. Verify that you can select various choices with both the mouse and the keyboard.

#### Proposed changelog entry for your changes:
* Added new setup wizard
